extensions/applicationinsights-analytics-js/src/applicationinsights-analytics-js.ts

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-export { ApplicationInsights } from "./JavaScriptSDK/ApplicationInsights";
+export { AnalyticsPlugin, AnalyticsPlugin as ApplicationInsights } from "./JavaScriptSDK/AnalyticsPlugin";
```

Comment:
Exporting with both the new and old names (for backward compatibility) -- internal extensions will be changed to use the new name.